src/pages/authentication/authentication.css

```diff
@@ -0,0 +1,23 @@
+.form-container {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+}
+.form {
+	max-width: 40rem;
+	width: 100%;
```

Comment:
Suggestion: give spacing between two CSS classes for readability.

Reply:
will try to format my code more properly.